app/views/users/show.html.erb

```diff
@@ -1,2 +1,14 @@
-<h1>This is a user</h1>
+<div class="users_container">
+<%= render partial: 'shared/card_user', locals: {user: @user} %>
+<p class="user_bio"><%= @user.bio%></p>
+<%#Add post for the user%>
+<% @posts.each do |post| %>
+<a href="<%=post.author_id%>/posts" class="user_link">
+<%= render partial: 'shared/card_post', locals: {post: post} %>
+</a>
+<%end%>
+<div class="btn"><button type="button" class="see_all_post">See all post</button></div>
+</div>
+
```

Comment:
- [ ] Please it would be better to add a default value 0 for post counters and comment counters and likes counters so we don't get this error when we try to save.
![image](https://user-images.githubusercontent.com/74744787/216399920-6cc11705-575d-4872-b768-2f6eb9cee6bf.png)

Reply:
Thanks for the comment @MeqdamAlqudah 
Can you please give me directives on how to go about that

Reply:
Hi Oswald, here is a [resource](https://jtway.co/how-to-setup-default-values-for-attributes-in-ruby-on-rails-dd1d2ba38b82) to help with that

Reply:
Please check the code again because I made sure I implemented the changes